Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -87,11 +87,20 @@ def bottle
     end
 
     return merge if args.merge?
+    ensure_formula_installed! "patchelf" unless OS.mac?
     ARGV.resolved_formulae.each do |f|
       bottle_formula f
     end
   end
 
+  def ensure_formula_installed!(formula)
+    return if Formula[formula].installed?
+    ohai "Installing #{formula}..."
+    safe_system HOMEBREW_BREW_FILE, "install", formula
```

Comment:
Seems weird to do a full shell out here?

Reply:
I cribbed that idiom from `Git.ensure_git_installed!`.
https://github.com/Homebrew/brew/blob/4e70a426d11e09459d2a96f2a52425a76bfba17b/Library/Homebrew/utils/git.rb#L55

Reply:
Ok 👍